extension/persistence/relational-jdbc/build.gradle.kts

```diff
@@ -17,7 +17,10 @@
  * under the License.
  */
 
-plugins { id("polaris-server") }
+plugins {
+  id("polaris-server")
+  alias(libs.plugins.jandex)
```

Comment:
nit: don't all modules get jandex by default?

Reply:
looks like a bunch of modules add it explicitly. We may clean it up if it comes as the default.

Reply:
My tests show that we still need to add it explicitly, otherwise, the `jandex.idx` will not be generated.

Reply:
right... I meant [this](https://github.com/apache/polaris/blob/6dc114924af120367b6b61f8c4ed2f7deface70e/build-logic/src/main/kotlin/polaris-java.gradle.kts#L181) but it apparently still expect the module to declare the jandex plugin.